README.md

```diff
@@ -407,16 +414,82 @@ foreach: '{{.step.prefixStrings.children | jsonmarshal}}'
 
 ## Extending µTask with plugins <a name="plugins"></a>
 
-TODO
+µTask is extensible with golang plugins compiled in *.so format. Two kinds of plugins exist:
```

Comment:
Add a link to Golang official plugin documentation?

Reply:
done